src/sys/boutexception.cxx

```diff
@@ -41,29 +45,7 @@ const char* BoutException::what() const throw() {
 }
 
 BoutRhsFail::BoutRhsFail(const char* s, ...)  : BoutException::BoutException(s) {
-  va_list ap;  // List of arguments
-
-  if(s == (const char*) NULL)
-    return;
-  
-  char buffer[BoutException::BUFFER_LEN];
-  va_start(ap, s);
-    vsnprintf(buffer, BoutException::BUFFER_LEN, s, ap);
-  va_end(ap);
-  
-  message.assign(buffer);
 }
 
 BoutIterationFail::BoutIterationFail(const char* s, ...) : BoutException::BoutException(s) {
-  va_list ap;  // List of arguments
```

Comment:
BoutIterationFail doesn't do anything ...
Well, currentl it does what the BoutException did once more - I don't think that is beneficial, thats why I removed the whole code.

Furthermore I couldn't find any documentation on va_list having any effect - by now I hide all the va_list/ap handling in the myvsnprint() macro ...